gtsam/base/Matrix.cpp

```diff
@@ -33,6 +34,7 @@
 #include <fstream>
 #include <limits>
 #include <iostream>
+#include <iostream>
```

Comment:
duplicate

Reply:
If you reviewed everything and you trust me to fix the above, approve??